### PR TITLE
Get files from server

### PR DESCRIPTION
**Pull request by yoouung**
- working on reconstructing
  - merge filelist and rules box in one page
- working on getting file list from server

### CODE REVIEW

**Comment by yoouung** [comment]
image

File: dt-demo-fe/src/Router.js [hunk 1]
```diff
@@ -17,10 +17,10 @@ const Router = () => {
                 <Route path="/" element={<Navigate replace to="/home" />} />
                 <Route path="/home" element={<Home />} />
                 <Route path="/filter" element={<Filter />}>
-                    <Route index element={<DemoUpload />} />
-                    <Route path="filelist" element={<DemoFileList />} />
-                    <Route path="rules" element={<DemoRules />} />
-                    <Route path="report" element={<DemoReport />} />
+                    {/* <Route index element={<UploadContainer />} /> */}
```
Comment (doorooful) [inline dt-demo-fe/src/Router.js:20]: Is this line necessary?

File: dt-demo-fe/src/pages/02_filter.js [hunk 2]
```diff
@@ -1,37 +1,22 @@
...
-    flex-direction: column;
+    flex-direction: row;
     align-items: center;
+    justify-content: center;
     height: 100vh;
+    width: 100%;
 `;
 
 const Filter = () => {
-    const [data, setData] = useState(null);
```
Comment (doorooful) [inline dt-demo-fe/src/pages/02_filter.js:15]: Thanks for removing test connection !

**Review by doorooful — APPROVED** [review]
Please find the comments!

File: dt-demo-fe/src/Router.js (hunk 1 above)
Comment (yoouung) [reply dt-demo-fe/src/Router.js:20]: this is not necessary, but I'll delete this line in next pr

File: dt-demo-fe/src/pages/02_filter.js (hunk 2 above)
Comment (yoouung) [reply dt-demo-fe/src/pages/02_filter.js:15]: thanks for comments!

**Pull request merged**